internal/repository/repository.go

```diff
@@ -0,0 +1,32 @@
+package repository
+
+import (
+	"log"
+	"time"
+
+	"github.com/pavkosykh/gallifrey/config"
+	"github.com/pavkosykh/gallifrey/internal/model"
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+)
+
+func StartTimeSerial(word string) {
```

Comment:
What do you think about use_cases?

I think there should be folder "use_cases" and inside of this files "start.go", "stop.go" and etc